test/features/sdk_flow.feature

```diff
@@ -113,3 +113,32 @@ Feature: SDK File Upload Tests
       | type |
       |      |
       | pdf  |
+
+  Scenario Outline: I can navigate to the previous screen
+    Given I navigate to the SDK
```

Comment:
These two lines (118 and 119) can be extracted to a separate Scenario, i.e "I can not see back button on welcome screen"